README.md

```diff
@@ -159,39 +159,38 @@ If you prefer to build only this package, just run:
 npm run build
 ```
 
-to simply build the package, or
+To build and keep watching for changes:
 
 ```shell script
 npm run build:dev
 ```
 
-to build and keep watching for changes.
-
-## Development with `Interfaces-Framework`
+## Development with `Interfaces-framework`
 
 To use `interfaces-core` for development (so using a checked-out version instead of the `npm` dependencies) some steps must be followed.
 
-Assuming that we have cloned [Interfaces-Framework](https://github.com/EthereansOS/Interfaces-Framework) in the same folder where we cloned this repo, we need to:
+Assuming that we have cloned [Interfaces-framework](https://github.com/EthereansOS/Interfaces-framework) in the same folder where we cloned this repo, we need to:
 
-- `npm install`
-- `npm link ../Interfaces-Framework/node_modules/react`
-- `npm build`
+- `npm install` (only if you don't already have the deps installed)
+- `npm link ../Interfaces-framework/node_modules/react`
+- `npm build` (or build:dev to develop in watch mode. See below)
 - `npm link`
 
-The first `npm link` links the `react` used by `Interfaces-Framework` to avoid [this problem](https://reactjs.org/warnings/invalid-hook-call-warning.html#duplicate-react).
+The first `npm link` links the `react` used by `Interfaces-framework` to avoid [this problem](https://reactjs.org/warnings/invalid-hook-call-warning.html#duplicate-react).
 
-Then, in `Interface-Framework`, for each packages/\*:
```

Comment:
This was wrong/outdated.

core and ui are hosted npm packages in the monorepo. They just need to be linked once, in the root